### ISSUE TITLE
[Bug Report][2.3.23] Accessibility: Textfield  error message doesn't get announced when the user tab into the field

### ISSUE THREAD

**Issue by tingrt** [vuetifyjs/vuetify#16467]
### Environment
**Vuetify Version:** 2.3.23
**Vue Version:** 2.6.10
**Browsers:** Chrome 108.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
1,Use the v-text-field 
2, Type the invalid input and make the error message is displayed 
3, Use the keyboard to tab to the field.
4, Using the NVDA (screen reader to test the accessibility).

### Expected Behavior
It should read the error message when the user tab to the field 

### Actual Behavior
NVDA doesn't read the error message

### Reproduction Link
[https://codepen.io/codeBelt/pen/OVjxJN](https://codepen.io/codeBelt/pen/OVjxJN)

### Other comments
We need to add aria-live="polite" on the error so it is announced without the user having to tab into the field

**Comment by deraw** [comment]
Hi!
@tingrt the link to your reproduction is an empty pen

I think the underlying issue is that the messages are not linked to the input. Adding the `aria-describedby` attribute with an ID for the message should fix the issue, but I don't see a way to do that when using the component.

**Comment by deraw** [comment]
There is an older issue for this, so it's a duplicate of #13874
